Gaming The Syllabus/Assets/Scripts/TooltipManager.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using TMPro;
+
+public class TooltipManager : MonoBehaviour
+{
+    public static TooltipManager _instance;
```

Comment:
Also yeah this is an alternate approach to the way I usually do singletons. If you use the base class and want a public instance to access you can just make a property: public TooltipManager Instance { get => _instance; }
Since the Singleton base class defines _instance